resources/yourls.rb

```diff
@@ -37,23 +41,39 @@
   end
 
   template "/var/www/#{new_resource.name}/yourls/user/config.php" do
+    source 'config.php.erb'
+    cookbook 'osl-php'
     variables(
-        db_username: new_resource.db_username,
-        db_password: new_resource.db_password,
-        db_name: new_resource.db_name,
+        cookiekey: new_resource.cookiekey,
         db_host: new_resource.db_host,
+        db_name: new_resource.db_name,
+        db_password: new_resource.db_password,
         db_prefix: new_resource.db_prefix,
+        db_username: new_resource.db_username,
         domain: new_resource.domain,
         language: new_resource.language,
-        unique_urls: new_resource.unique_urls,
         private: new_resource.private,
-        cookiekey: new_resource.cookiekey,
-        user_passwords: new_resource.user_passwords,
+        reserved_urls: new_resource.reserved_urls,
+        unique_urls: new_resource.unique_urls,
         url_convert: new_resource.url_convert,
-        reserved_urls: new_resource.reserved_urls
+        user_passwords: new_resource.user_passwords
       )
-    sensitive true
   end
 
-  osl_php_install 'default'
+  file "/var/www/#{new_resource.name}/yourls/.htaccess" do
```

Comment:
Please use a `cookbook_file` instead. We only do something like this if it's something small and simple.